### PR TITLE
Add D2D token to Base

### PR DESCRIPTION
**Pull request by fabianschu**

### CODE REVIEW

File: data/D2D/data.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "tokens": {
+    "ethereum": {
+      "address": "0x43D4A3cd90ddD2F8f4f693170C9c8098163502ad"
+    },
+    "base": {
+      "address": "0x91e1d2b9b0d9afcf1f024df09a68799fb200bc63"
+    },
+    "optimism": {
+      "address": "0x03e5b5930f3a42b56af4a982b10e60957bdd2f61"
+    },
```
Comment (roberto-bayardo) [inline data/D2D/data.json:17]: json failing to validate, I think you need to drop this comma

Comment (roninjin10) [reply data/D2D/data.json:17]:
```suggestion
    }
```

**Comment by roberto-bayardo** [comment]
ping @fabianschu

**Comment by fabianschu** [comment]
@roberto-bayardo thanks for the ping, I dropped the comma.

**Review by roberto-bayardo — APPROVED** [review]

**Pull request merged**